app/src/main/java/com/larryhsiao/nyx/ImportFromBackup.java

```diff
@@ -0,0 +1,98 @@
+package com.larryhsiao.nyx;
+
+import android.content.Context;
+import android.os.Environment;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.larryhsiao.nyx.attachments.NewAttachment;
+import com.larryhsiao.nyx.jots.*;
+import com.silverhetch.clotho.Action;
+import com.silverhetch.clotho.Source;
+import com.silverhetch.clotho.file.FileText;
+import com.silverhetch.clotho.source.ConstSource;
+
+import java.io.File;
+import java.sql.Connection;
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * Action for importing backup.
+ */
+public class ImportFromBackup implements Action {
+    private final Source<Connection> db;
+    private final Context context;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'context' could be replaced by a local variable.](https://app.codacy.com/manual/LarryHsiao/Nyx/pullRequest?prid=5060172)